src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
-package core.basesyntax;
-
-public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
-    }
-}
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class SalaryInfo {
+
+    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Report for period ").append(dateFrom).append(" - ").append(dateTo);
+        for (int i = 0; i < names.length; i++) {
+            int totalSalory = 0;
+            totalSalory = getTotalSalory(names[i],data,dateFrom,dateTo);
+            builder.append("\n").append(names[i]);
+            builder.append(" - ").append(totalSalory);
+        }
+        return builder.toString();
+    }
+
+    public int getTotalSalory(String name, String[] data, String dateFrom, String dateTo) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");

```

Comment:
![Screenshot 2022-06-20 at 11 46 38](https://user-images.githubusercontent.com/60258373/174562806-f637dc6c-2382-4d04-8fff-06a79a620a1b.png)
Before you create PR with your homework, you need to go through the checklist under the task, and correct all the points described there. The mentor will not check the work until the checklist items are corrected